resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -201,6 +203,16 @@ public DnsNameResolverBuilder authoritativeDnsServerCache(AuthoritativeDnsServer
         return this;
     }
 
+    /**
+     * Sets the udp client listen local Address
```

Comment:
```suggestion
     * Configure the address that will be used to bind too. If `null` the default will be used.
```

Reply:
Sorry, I don't know I can commit the suggestion, Fixed